packages/sync/api-report.md

```diff
@@ -41,6 +45,7 @@ export interface UseSyncDemoOptions {
 // @public
 export interface UseSyncOptions {
     assets: TLAssetStore;
+    getUserPresence?(store: TLStore, user: TLSyncUserInfo): null | TLPresenceStateInfo;
```

Comment:
```suggestion
    getUserPresence?(store: TLStore, user: TLPresenceUserInfo): null | TLPresenceStateInfo;
```
@SomeHats I got type error when using the options:

### Sample code

```ts
  const store = useSync({
    uri: `${HOST}/connect/${roomId}`,
    getUserPresence(store, user) {
      const defaults = getDefaultUserPresence(store, user)
      return defaults
    },
  })
```

### error:
```vscode
Argument of type 'TLSyncUserInfo' is not assignable to parameter of type 'TLPresenceUserInfo'.
  Types of property 'color' are incompatible.
    Type 'string | null | undefined' is not assignable to type 'string'.
      Type 'undefined' is not assignable to type 'string'.ts(2345)
```